README.md

```diff
@@ -269,6 +269,7 @@ mjpegScalingFactor | int | The percentage value used to apply downscaling on the
 mjpegServerScreenshotQuality | int | The percentage value used to apply lossy JPEG compression on the screenshots generated by the MJPEG screenshots broadcaster. Must be in range 1..100. `50` is by default, which means that screenshots are compressed to the half of their original quality.
 mjpegBilinearFiltering | boolean | Controls whether (`true`) or not (`false`, the default value) to apply bilinear filtering to MJPEG screenshots broadcaster resize algorithm. Enabling this flag may improve the quality of the resulting scaled bitmap, but may introduce a small performance hit.
 useResourcesForOrientationDetection | boolean | Defines the strategy used by UiAutomator2 server to detect the original device orientation. By default (`false` value) the server uses device rotation value for this purpose. Although, this approach may not work for some devices and a portrait orientation may erroneously be detected as the landscape one (and vice versa). In such case it makes sense to play with this setting.
+enforceXPath1 | boolean | Since UiAutomator2 driver version `4.25.0` XPath2 is set as the default interpreter for the corresponding element locators. This interpreter is based on [Psychopath XPath2](https://wiki.eclipse.org/PsychoPathXPathProcessor) implementation, which is now a part of the Eclipse foundation. In most of the cases XPath1 locators are also valid XPath2 locators, so there should be no issues while locating elements. Although, since the XPath2 standard is much more advanced and complicated in comparison to the previous version, some [issues](https://github.com/appium/appium/issues/16142) are possible, which cannot be fixed easily, as we depend on the third-party library mentioned above. Then try to workaround such issues by enforcing XPath1 usage (whose implementation is a part of the Android platform itself) and assigning this setting to `true`. Note, this setting is actually applied at the time when the element lookup by XPath is executed, so you could switch it on or off whenever needed throughout your automated testing session.
```

Comment:
Should we address this as _not recommended as possible_?

Reply:
I don't see a need for this. If someone wants to still use XPath1 then why should we not advice that?

Reply:
I thought we'll support XPath2 only in the future.
Then, addressing something such as _xpath2 will be a primary way_ or _recommended way_ could reduce confusion to update XPath 1 to XPath 2 (if needed) on users' side since this capability could think UIA2 will support both XPath 1 and 2. So users do not need to update their xpath to 2 when they get an error in XPath2 without trying to update XPath to 2.

Or probably when we drop xpath1, we need to add a deprecation message and support both for a while with this capability(?)

Reply:
Makes sense. Will update the description

Reply:
thanks